plaidml/edsl/ffi.cc

```diff
@@ -973,6 +976,10 @@ plaidml_program* plaidml_program_evaluate(  //
       }
       mutations.updates.emplace(ProgramUpdate{src_updates[i]->value, dst_updates[i]->value});
     }
+
+    mutations.set_floatx(to_string(static_cast<plaidml::DType>(floatx)));
+    mutations.set_intx(to_string(static_cast<plaidml::DType>(intx)));
```

Comment:
I don't think we should ever convert to a string unless we are: 1) printing or 2) parsing the type. But if we're just passing the type along thru different layers, we should depend on the enum (not the string value, strings just have a higher cost compared to a simple integer/aka enum value)